components/delivery/provider/index.js

```diff
@@ -1,28 +1,50 @@
 import Packetery, { validationScheme as packeteryValidationScheme } from "./providers/packetery";
+import get from "lodash/get";
 
 export const PACKETERY_TYPE = 1;
 
-export const getValidationScheme = (deliveryProviderType = null) => {
+/*
+getValidationScheme can handle options
+options.text : {
+    Packetery : {
+        validationScheme : "Vyberte odberné miesto"
+    }
+}
+ */
+export const getValidationScheme = (deliveryProviderType = null, options) => {
     if (!deliveryProviderType) {
         return null
     }
 
     switch (deliveryProviderType) {
         case PACKETERY_TYPE:
-            return packeteryValidationScheme;
+            return packeteryValidationScheme(get(options, 'text.Packetery.validationScheme', "Vyberte odberné miesto"));
```

Comment:
@matejkopcik zda sa mi, alebo tu pracujes s `options?.text?.Packetery.....` a v `Packetery` komponente uz pouzivas `options?.Packetery?.text...`. nezjednotime to? alebo je za tym nejaky zamer, ktory mi usiel? Dakujem

Reply:
samotna getValidationScheme dostala options, kedze tie options mozu neskor sa rozsirovat tak pod klucom text su evidovane texty pre cely getValidationScheme. Samotna packeteryValidationScheme ma v sebe jeden argument a to text na vypis validacnej hlasky, preto do getValidationScheme => options.text.Packetery.validationScheme sa dava ten text. Vychadzal z toho ako je nastaveny DeliveryProvider, ktory v options.Packetery berie options cele do Packetery, ale kedze vo validacnych scheme je len jeden text tak som to takto nastavil

Reply:
ok. ide mi vlastne len o to, ci tam nebudeme namiesto:
```
options: {
   text: {
      Packetery: {
          validationScheme: 'validacna hlaska'
      }
   }
}
```
posielat 
```
options: {
   Packetery: {
      text: {
          validationScheme:  'validacna hlaska'
      }
   }
}
```
lebo ak to budeme chciet rozsirit o nieco dalsie, napr. nejaku externu hodnotu (nie text), tak aby to bolo vsetko pod klucom Packetery, ktory mu tam vies poslat ako celok
```
options: {
   Packetery: {
     text: {
         validationScheme:   'validacna hlaska',
     },
     field2: 'hodnota fieldu 2',
     field3: 'hodnota fieldu 3'
   }
}
```
a nie roztrusene nasledovne:
```
options: {
   text: {
        Packetery: 
            validationScheme : 'validacna hlaska',
   },
   
   field2: {
       Packetery: 'hodnota fieldu 2',
   }
   field3: {
       Packetery: 'hodnota fieldu 3'
   }
}
```
potom budes musiet posielat jednotlive option postupne ako:
```
packeteryValidationScheme(
    get(options, 'text.Packetery.validationScheme'),
    get(options, 'field2.Packetery'),
    get(options, 'field3.Packetery')
);
```
kdezto ked to bude vsetko pod klucom `options?.Packetery` tak to vies poslat ako jeden argument funckie:
```
packeteryValidationScheme(get(options, 'Packetery'));
```
kde si uz priamo vo funkcii `packeteryValidationScheme` vytiahnes z options co budes potrebovat a kde budes potrebovat. a ak rozsiris options o nieco nove, tak to nebudes musiet menit/doplnat aj v `components/delivery/provider/index.js`, ale len v tom danom provideri `components/delivery/provider/providers/packetery.js`